perma_web/perma/templates/docs/docs_menu.html

```diff
@@ -41,7 +41,7 @@
           {% else %}
           <a class="overview-module-content" href="{% url 'docs_libraries' %}">
           {% endif %}
-            <strong class="overview-module-title">For Libraries</strong>
+            <strong class="overview-module-title">For Registrars</strong>
```

Comment:
Two extra spaces, I think?

Reply:
I think this one is in line with the rest of the <strong> tags? unless I'm missing something